uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/AppContent.kt

```diff
@@ -37,15 +40,14 @@ fun AppContent(
             drawerState.open()
         }
     }
+    val actions = remember(navController) { AppActions(navController) }
     ModalDrawer(
         modifier = modifier,
         drawerState = drawerState,
         drawerShape = MaterialTheme.shapes.large.copy(all = CornerSize(0.dp)),
         drawerContent = {
             DrawerContent { route ->
-                try {
-                    navController.navigate(route)
-                } finally {
+                actions.onSelectDrawerItem(route) {
```

Comment:
Thank you for leaving the original code! 
I don't think I need try catch anymore, so maybe I can write it like this

```kotlin
actions.onSelectDrawerItem(route)
coroutineScope.launch {
     drawerState.close()
}
```

Reply:
Thanks a lot for your review!

> I don't think I need try catch anymore

You'are right. It's my wrong!
I simplified this part!!
https://github.com/DroidKaigi/conference-app-2021/pull/161/commits/95eab98f8a0de26aa3aa15ef75eee991fbe31b69